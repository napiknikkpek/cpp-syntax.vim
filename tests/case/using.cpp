typedef Foo foo;
using F = int;
using Foo = Bar;
using F;
using foo::Foo;
