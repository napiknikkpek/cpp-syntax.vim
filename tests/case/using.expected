aaaaaaa.ttt.xxx.
aaaaa.t...zzz.
aaaaa.ttt...xxx.
aaaaa.t.
aaaaa.ttt..xxx.
